modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb

```diff
@@ -16,10 +16,12 @@ def type
           validate_request!(ClaimsApi::V2::ParamsValidation::IntentToFile)
 
           type = get_bgs_type(params)
-          response = bgs_service.intent_to_file.find_intent_to_file_by_ptcpnt_id_itf_type_cd(
+          response = local_bgs_service.find_intent_to_file_by_ptcpnt_id_itf_type_cd(
             target_veteran.participant_id,
             type
           )
+          message = "No active '#{params[:type]}' intent to file found."
+          raise ::Common::Exceptions::ResourceNotFound.new(detail: message) if response.blank?
 
           response = [response] unless response.is_a?(Array)
```

Comment:
I tested by modifying parsed_response to always return an empty array, and we definitely hit this. Going to leave as-is because it saves us from performing unnecessary logic on an empty array.